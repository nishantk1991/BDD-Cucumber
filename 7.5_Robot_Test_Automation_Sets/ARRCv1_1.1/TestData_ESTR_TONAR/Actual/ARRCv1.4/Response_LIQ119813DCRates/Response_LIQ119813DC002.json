{
  "comment": " The most recent published ESTR RFR rate is available as-of: 2020-05-19. Computed time : 2021-11-22T11:43:01+00:00. Computed time is the time stamp when the calculation was completed.",
  "lastUpdatedFromSource": "2021-10-29T00:00:00+00:00",
  "legalDisclaimer": "Use of euro short-term rate (€STR) is subject to the ECB’s €STR disclaimer under https://www.ecb.europa.eu/stats/financial_markets_and_interest_rates/euro_short-term_rate/html/index.en.html#disclaimer.",
  "requestParameters": {
    "interestPeriodStartDate": "2020-05-03",
    "interestPeriodEndDate": "2020-05-31",
    "riskFreeRate": "ESTR",
    "riskFreeRateFloor": 0.22348,
    "lookBackDays": 4,
    "lockoutDays": 0,
    "observationPeriodShift": true,
    "dayCountConvention": "ACT/360",
    "rateRoundingDecimalPoints": 12,
    "showDailyDetails": true,
    "calculateTillDate": "2020-05-16"
  },
  "dailyRateSummary": [
    {
      "interestPeriodDate": "2020-05-15",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-05-11",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.06,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000054097984,
      "dailyCompoundedRate": 0.223492089818
    },
    {
      "interestPeriodDate": "2020-05-14",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-08",
      "observationPeriodDays": 3,
      "publishedRiskFreeRate": 0.06,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.00012948089,
      "dailyCompoundedRate": 0.223508936389
    },
    {
      "interestPeriodDate": "2020-05-13",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-07",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.05,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000062079396,
      "dailyCompoundedRate": 0.223493873504
    },
    {
      "interestPeriodDate": "2020-05-12",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-06",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.05,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000055871272,
      "dailyCompoundedRate": 0.223492486112
    },
    {
      "interestPeriodDate": "2020-05-11",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-05",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.05,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000049663186,
      "dailyCompoundedRate": 0.223491098729
    },
    {
      "interestPeriodDate": "2020-05-08",
      "interestPeriodDays": 3,
      "observationPeriodDate": "2020-05-04",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.05,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000025127032,
      "dailyCompoundedRate": 0.223485615389
    },
    {
      "interestPeriodDate": "2020-05-07",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-01",
      "observationPeriodDays": 3,
      "publishedRiskFreeRate": 0.03,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.00004256807,
      "dailyCompoundedRate": 0.223489513112
    },
    {
      "interestPeriodDate": "2020-05-06",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-30",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.04,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000018623449,
      "dailyCompoundedRate": 0.223484161968
    },
    {
      "interestPeriodDate": "2020-05-05",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-29",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.01,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000012415594,
      "dailyCompoundedRate": 0.223482774637
    },
    {
      "interestPeriodDate": "2020-05-04",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-28",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.01,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.000006207778,
      "dailyCompoundedRate": 0.223481387314
    },
    {
      "interestPeriodDate": "2020-05-03",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-04-28",
      "observationPeriodDays": 1,
      "publishedRiskFreeRate": 0.01,
      "rateApplied": 0.22348,
      "riskFreeRateFloorApplied": true,
      "compoundingFactor": 1.0,
      "dailyCompoundedRate": 0.22348
    }
  ]
}