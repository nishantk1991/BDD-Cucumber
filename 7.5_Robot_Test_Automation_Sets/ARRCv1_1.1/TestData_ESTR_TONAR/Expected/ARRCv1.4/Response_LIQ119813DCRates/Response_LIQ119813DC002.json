{
    "comment": " Computed time : 2021-03-31T07:16:14+00:00. Computed time is the time stamp when the calculation was completed.",
    "lastUpdatedFromSource": "2020-08-05T09:45:12+00:00",
    "legalDisclaimer": "Use subject to New York Fed Terms of Use for Select Rate Data.  New York Fed has no liability for your use.  Refer the following link for more details: https://www.newyorkfed.org/markets/reference-rates-terms-of-use",
    "requestParameters": {
        "interestPeriodStartDate": "2020-05-03",
        "interestPeriodEndDate": "2020-05-31",
        "riskFreeRate": "ESTR",
        "riskFreeRateFloor": 0.22348,
        "lookBackDays": 4,
        "lockoutDays": 0,
        "observationPeriodShift": true,
        "dayCountConvention": "ACT/360",
        "rateRoundingDecimalPoints": 12,
        "showDailyDetails": true,
        "calculateTillDate": "2020-05-16"
    },
    "dailyRateSummary": [
        {
            "interestPeriodDate": "2020-05-15",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-05-11",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.06,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000054097984,
            "dailyCompoundedRate": 0.223492089818
        },
        {
            "interestPeriodDate": "2020-05-14",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-08",
            "observationPeriodDays": 3,
            "publishedRiskFreeRate": 0.06,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000129480890,
            "dailyCompoundedRate": 0.223508936389
        },
        {
            "interestPeriodDate": "2020-05-13",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-07",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.05,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000062079396,
            "dailyCompoundedRate": 0.223493873504
        },
        {
            "interestPeriodDate": "2020-05-12",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-06",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.05,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000055871272,
            "dailyCompoundedRate": 0.223492486112
        },
        {
            "interestPeriodDate": "2020-05-11",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-05",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.05,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000049663186,
            "dailyCompoundedRate": 0.223491098729
        },
        {
            "interestPeriodDate": "2020-05-08",
            "interestPeriodDays": 3,
            "observationPeriodDate": "2020-05-04",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.05,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000025127032,
            "dailyCompoundedRate": 0.223485615389
        },
        {
            "interestPeriodDate": "2020-05-07",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-01",
            "observationPeriodDays": 3,
            "publishedRiskFreeRate": 0.03,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000042568070,
            "dailyCompoundedRate": 0.223489513112
        },
        {
            "interestPeriodDate": "2020-05-06",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-30",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.04,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000018623449,
            "dailyCompoundedRate": 0.223484161968
        },
        {
            "interestPeriodDate": "2020-05-05",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-29",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.01,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000012415594,
            "dailyCompoundedRate": 0.223482774637
        },
        {
            "interestPeriodDate": "2020-05-04",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-28",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.01,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000006207778,
            "dailyCompoundedRate": 0.223481387314
        },
        {
            "interestPeriodDate": "2020-05-03",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-04-28",
            "observationPeriodDays": 1,
            "publishedRiskFreeRate": 0.01,
            "rateApplied": 0.22348,
            "riskFreeRateFloorApplied": true,
            "compoundingFactor": 1.000000000000,
            "dailyCompoundedRate": 0.223480000000
        }
    ]
}